README.md

```diff
@@ -10,20 +10,20 @@ urlFragment: Azure-Time-Series-Insights
 
 # Azure Time Series Insights Samples
 
-[![TSI: GA](https://img.shields.io/badge/TSI-GA-blue.svg)](https://docs.microsoft.com/azure/time-series-insights/time-series-insights-overview) [![TSI: Preview](https://img.shields.io/badge/TSI-Preview-blue.svg)](https://docs.microsoft.com/azure/time-series-insights/time-series-insights-update-overview) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![Contribute](https://img.shields.io/badge/PR%27s-welcome-brightgreen.svg)](CONTRIBUTING.md)
+[![TSI: Gen1](https://img.shields.io/badge/TSI-Gen1-blue.svg)](https://docs.microsoft.com/azure/time-series-insights/time-series-insights-overview) [![TSI: Gen2](https://img.shields.io/badge/TSI-Gen2-blue.svg)](https://docs.microsoft.com/azure/time-series-insights/overview-what-is-tsi) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![Contribute](https://img.shields.io/badge/PR%27s-welcome-brightgreen.svg)](CONTRIBUTING.md)
 
 These are samples that use the [Azure Time Series Insights](https://azure.microsoft.com/services/time-series-insights/) data plane APIs.
 
 ## Samples
 
-### TSI Preview
-* [C# Sample App](https://github.com/Azure-Samples/Azure-Time-Series-Insights/tree/master/csharp-tsi-preview-sample)
+### TSI Gen2
+* [C# Sample App](https://github.com/Azure-Samples/Azure-Time-Series-Insights/tree/master/gen2-sample/csharp-tsi-gen2-sample)
 
-### TSI General Availability
+### TSI Gen1
 
-* [C# Sample App](https://github.com/Azure-Samples/Azure-Time-Series-Insights/tree/master/csharp-tsi-ga-sample)
-* [C# Sample App using MSAL.NET](https://github.com/Azure-Samples/Azure-Time-Series-Insights/tree/master/csharp-tsi-msal-ga-sample)
-* [JavaScript Sample App](https://github.com/Azure-Samples/Azure-Time-Series-Insights/tree/master/javascript-tsi-ga-sample)
+* [C# Sample App](https://github.com/Azure-Samples/Azure-Time-Series-Insights/tree/master/gen1-sample/csharp-tsi-gen2-sample)
+* [C# Sample App using MSAL.NET](https://github.com/Azure-Samples/Azure-Time-Series-Insights/tree/master/gen1-sample/csharp-tsi-msal-gen1-sample)
```

Comment:
It appears the convention is of the form: [language]-tsi-[genX]-[additional_context]-sample. 
We should make csharp-tsi-msal-gen1-sample as csharp-tsi-gen1-msal-sample to keep consistent project naming convention